python/snewpy/scripts/snewpy_to_snewpdag.py

```diff
@@ -0,0 +1,67 @@
+
+from argparse import ArgumentParser
+from snewpy import to_snowglobes
+from snewpy import run_snowglobes
+from snewpy import from_snowglobes
+import tarfile
+import numpy as np
+import os
+from astropy.io import ascii
+from astropy import units as u
+import matplotlib.pyplot as plt
+
+SNOwGLoBES_path = "/location/of/snowglobes/" #where snowglobes is located
+SNEWPY_models_base = "/location/of/models/" #where models (aka input for to_snowglobes) is located
+
+#set distance in kpc
+distance=10
+
+#set SNOwGLoBES detector to use
+detector = "icecube"
+
+#set SNEWPY model type and filename
+modeltype = 'Tamborra_2014'
+modeldir = SNEWPY_models_base+"/"+modeltype+"/"
+model = 's20.0c_3D_dir1'
+filename = model
+
+#set desired oscillation prescription
+transformation = 'AdiabaticMSW_NMO'
+
+#to_snowglobes creates a tarred file of snowglobes fluences
+#this is stored in the model types directory in the snewpy/models
+#the full path is returned from to_snowglobes
+outfile = modeltype+"_"+model+"_"+transformation
+
+#Specify sequence of time intervals, one fluence
+#file is made for each elements with dt=tstart[i]-tend[i]
+window_tstart = 0.001
+window_tend = 0.331
+window_bins = 330
+tstart = np.linspace(window_tstart,window_tend,window_bins,endpoint=False)*u.s
+tend = tstart + (window_tend-window_tstart)/window_bins*u.s
+tmid = (tstart+tend)*0.5
+
+#call to_snowglobes
+#There are two options, here we use generate_fluence
+print("Preparing fluences...")
+tarredfile = to_snowglobes.generate_fluence(modeldir, filename, modeltype, transformation, distance, outfile,tstart,tend)
+print("Done fluences...")
+
+print("Running snowglobes...")
+#now run snowglobes, this will loop over all the fluence files in `tarredfile`
+run_snowglobes.go(SNOwGLoBES_path, modeldir, tarredfile, detector_input=detector,verbose=False)
+print("Done snowglobes...")
+
+#now collate results of output of snowglobes
+print("Collating...")
+tables = from_snowglobes.collate(SNOwGLoBES_path, modeldir, tarredfile, detector_input=detector,skip_plots=True,return_tables=True,verbose=False)
+
+#read results from snowglobes and put lightcurve in output file for snewpdag
+fout = open("/path/to/output/snewpy_output_"+detector+"_"+modeltype+"_"+filename+"_1msbin.txt", "a")
```

Comment:
It may be nicer to move `/path/to/output/` into a separate variable that’s defined at the top of the file, right after `SNOwGLoBES_path` and `SNEWPY_models_base`; that way, users will find all the paths they need to adjust in one place.

Reply:
yes, agree!